arviz/plots/ecdfplot.py

```diff
@@ -0,0 +1,107 @@
+"""Plot ecdf and ecdf-difference plot with confidence bands."""
+import numpy as np
+
+from arviz.rcparams import rcParams
+from arviz.plots.plot_utils import get_plotting_function
+
+
+def plot_ecdf(
+    values,
+    values2=None,
+    distribution=None,
+    difference=False,
+    pit=False,
+    confidence_bands=False,
+    granularity=100,
+    num_trials=500,
+    alpha=0.95,
+    figsize=None,
+    ecdf_fill=True,
+    ax=None,
+    show=None,
+    backend=None,
+    backend_kwargs=None,
+    **kwargs
+):
+    """Plot ECDF and ECDF-Difference Plot with Confidence bands.
+
+    Parameters
+    ----------
+    values : array-like
+        Values to plot from an unknown continuous distribution
+    values2 : array-like, optional
+        Values to compare to the original sample
```

Comment:
So by default the `confidence_bands` argument is false so that when you are doing just plot_ecdf(x,y) it plots the normal ecdf plot of x rather than comparing it with y and plotting the confidence intervals. So should I set the default value of `confidence_bands` to true?

Reply:
Ah, so you're saying that when `confidence_bands` is `True`, the confidence band is determined by comparing `values` and `values2`? What does this correspond to in the paper?

Reply:
So in the paper, they had only plotted ecdf for PIT of the sample, so for calculation of PIT they have used either the distribution or the comparison sample which I have also done. But in my case I am also plotting ecdf for the sample, so for finding the evaluation points for the uniform distribution while calculating gamma I have used the cdf(x) where x is equally spaced points between min and max. So basically if we are provided the sample and cdf or the sample and comparision sample then if `confidence_bands` is false then there is no use of cdf or comparison sample and a normal ecdf plot is plotted for the sample.